packages/react/src/page-alert/PageAlert.tsx

```diff
@@ -77,7 +81,7 @@ export const PageAlert = forwardRef<HTMLDivElement, PageAlertProps>(
 				ref={ref}
 				role={role}
 				rounded
-				tabIndex={tabIndex || (focusOnMount ? -1 : undefined)}
+				tabIndex={tabIndex || (focusOnMount || focusOnUpdate ? -1 : undefined)}
```

Comment:
Why are these 2 in parens? What's the significance of that?

Reply:
Actually, it should be `tabIndex ?? ...` (rather than `||`).

The problems to solve are:
1. A value set for `tabIndex` should be used first.
2. If `tabIndex` is `undefined` and there's a value for `focusOnMount` or `focusOnUpdate` then we want `-1`.

- [ ] use `??`